dali/imgcodec/parsers/bmp.cc

```diff
@@ -13,18 +13,154 @@
 // limitations under the License.
 
 #include "dali/imgcodec/parsers/bmp.h"
+#include "dali/core/byte_io.h"
+#include "dali/core/format.h"
+#include "dali/core/small_vector.h"
 
 namespace dali {
 namespace imgcodec {
 
+namespace  {
+
+enum BmpCompressionType {
+  BMP_COMPRESSION_RGB       = 0,
+  BMP_COMPRESSION_RLE8      = 1,
+  BMP_COMPRESSION_RLE4      = 2,
+  BMP_COMPRESSION_BITFIELDS = 3
+};
+
+template <typename T>
+T ConsumeValue(const uint8_t*& ptr) {
```

Comment:
Not used.

Reply:
Can you also enable the testcase that parses BMP?

Reply:
Done.